### DISCUSSION TITLE
May 19 2021 Full Team Meeting minutes

### DISCUSSION THREAD

**Discussion by mli1** [SafetyGraphics/safetyGraphics#567] [Meetings]
# Attendees
18 attendees in total @xni7 @JimBuchanan @PHayashi @DOBrien1  @spduke @hendricksonb @MayaGans @mli1, Ted Guo, Nilesh Patel @wal615 @sibyllwangxxxx  Greg Ball, Asli Memisoglu, Danae Williams @ Mike Colopy _(Note from Mengchun: sorry I can't tell who is missing, if you joined the meeting but your name is missing, please kindly let me know)_

# Agenda/Notes

1. Agenda item 1-- Tech update: V2.0 alpha testing is about ready to be kicked off. All the tech members and core team members will be invited for alpha testing. Anyone else who is also interested please reach out to Jeremy/Xiao. Also, please refer to post #538 

2. Agenda item 2-- Team building prep. _Post-meeting note: A successful team building event was held on May 21. 16 members joined the team building session, team members did a brief introduction to everyone else, before they were divided into three breakout rooms. Within each breakout room, team members introduced 2 facts and 1 lie about themselves and asked the rest to guess which is false; team members also shared 1-2 important pictures of themselves and explained why. Then all the members gathered together in the main session with highlight sharing.

3. Agenda item 3-- Subgroup update: 

- AE volcano subgroup: A subgroup discussion was conducted earlier on deliverables. Team to work on the clinical workflow with input from statisticians
- QT subgroup: A joint meeting with CSRC was held on May 5 2021, with great outcome (that CSRC was very interested on this joint journey with ISG). A survey was conducted within the CSRC+ISG joint group, on what are the areas to collaborate/ objectives/deliverables. 
- PHUSE collaboration: Greg Ball introduced that there will be a multiple-working-group-brainstorming session to be held on Jun 10, in the hope that multiple working groups where there are shared interests and goals, common ground can be found for better collaboration and synergy. 
- Launch readiness subgroup: This idea has not been discussed at leadership council meeting yet. However the idea was from one of the previous ISG meetings that if we should consider form an internal launch readiness team to help promoting the deliverables from the taskforce/working group. If so, should this team be part of ISG, or should it be part of the outreach and communication team? Will update the team when this is discussed at leadership council meeting
- PSI and R in Pharma--- Looking for speaker from the team to present our good work. Team member to reach out to Xiao/Jeremy if interested.

_(Extra note from Mengchun: A core team meeting was held on May 18, 2021, to discuss the agenda items for this May 19 meeting. No separate meeting minutes is provided giving everything covered there is now included in this note)_

**Comment by aslimem** [comment]
@mli1 Here I am.

**Reply by mli1** [reply]
This is so strange... I can @ you here, but I am not able to @ you in the other meeting minutes. @jwildfire @xni7 do you know what is going on? When I was trying to @ Asli in today's meeting minutes (Jun 16), I could not find her name at all. But I can @ here, do you know why?

**Reply by xni7** [reply]
it seems @aslimem is not part of the safetyGraphics org yet. Just sent an invite, @aslimem can you accept?

**Comment by nilesh** [comment]
Hey guys please untag me from this thread. I am not the right person
